docs/api/Model.md

```diff
@@ -1,26 +1,21 @@
 # Model
 
-The Model object represents one DynamoDB table. It takes in both a name and a schema and has methods to retrieve, and save items in the database.
+The Model object represents a table in DynamoDB. It takes in both a name and a schema and has methods to retrieve, and save items in the database.
 
 ## new dynamoose.Model(name, schema[, config])
 
-This method is the basic entry point for creating a model in Dynamoose. When you call this method a new model is created, and it returns a Document initializer that you can use to create instances of the given model.
+This method is the basic entry point for creating a model in Dynamoose. When you call this method a new model is created, and it returns an initializer that you can use to create instances of the given model.
 
-The `schema` parameter can either be an object OR a Schema instance. If you pass in an object for the `schema` parameter it will create a Schema instance for you automatically.
+The `schema` parameter can either be an object OR a Schema instance, see [Schema](Schema.md) for details. If you pass in an object for the `schema` parameter it will create a Schema instance for you automatically.
 
 ```js
-const dynamoose = require("dynamoose");
```

Comment:
Why was this removed? Model is a common starting point for Dynamoose. I think it's important to understand how this is happening.

Reply:
It is already part of General.md and i didnt see the require anywhere else. So i thought it would consistent to have it nowhere. For me schema would be the entry point.

Reply:
Adding this back to only the Model and Schema initialization sections.